docs/dev-guide/integration/rest-api/repo/cud.rst

```diff
@@ -159,7 +159,7 @@ importer type will support a different set of values relevant to how it function
         There is an optional importer configuration parameter that is usable
         on all importer plugin types called ``force_full``. If this parameter is set,
         a full re-sync will be forced.
-        More details are `here <http://pulp.readthedocs.org/en/latest/user-guide/release-notes/2.10.x.html#pulp-2-10-0>`_.
+        More details could be found in `the release notes for Pulp 2.10.0 <http://docs.pulpproject.org/user-guide/release-notes/2.10.x.html#pulp-2-10-0>`_.
```

Comment:
Could maybe put a newline after 'Pulp 2.10.0' to make the line less long. Up to you.